### PR TITLE
Google Tag Manager to Docs

### PR DESCRIPTION
**Pull request by Go-Fred**
I'm not sure that it's the good directory and that it's belong to the doc. Can you check please? 
@gozman @jpjoyal @alavers

### CODE REVIEW

File: source/includes/integrations/_google_tag_manager.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## Google Tag Manager
+
+You can load Smooch's web widget through Google Tag Manager. Simply connect to your Google Tag Manager account, go inside your container and follow the guide below. 
+
+### 1. Create a new Tag and give it a cute name like "Smooch"
+
+On your Google Tag Manager dashboard, create a new tag and rename it "Smooch".
+
+![GTM New Tag](/images/gtm_new_tag.png)
```
Comment (jpjoyal) [inline source/includes/integrations/_google_tag_manager.md]: I would have this screenshot zoomed in the name and have the Title show in rename mode

File: source/android/index.md [hunk 2]
```diff
@@ -47,6 +47,8 @@ includes:
   - integrations/stripe_notice_android
   - integrations/shopify
   - integrations/connecting_crm
+  - integrations/google_tag_manager
```
Comment (jpjoyal) [inline source/android/index.md]: I would move this under "Adding Smooch to your site"

File: source/includes/integrations/_google_tag_manager.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+## Google Tag Manager
+
+You can load Smooch's web widget through Google Tag Manager. Simply connect to your Google Tag Manager account, go inside your container and follow the guide below. 
+
+### 1. Create a new Tag and give it a cute name like "Smooch"
```
Comment (dannytranlx) [inline source/includes/integrations/_google_tag_manager.md]: > give it a cute name like "Smooch"

We will have to talk with our copywriter about that :joy:

**Comment by Go-Fred** [comment]
@jpjoyal I updated with your comments. 
@dannytranlx 😂 - I hope my PR will not get rejected because of that...

**Comment by jpjoyal** [comment]
image

**Comment by jpjoyal** [comment]
you are missing images

**Comment by Go-Fred** [comment]
They are on my local dir...

**Comment by jpjoyal** [comment]
:+1:

File: source/includes/_adding_smooch_js.md [hunk 4]
```diff
@@ -52,3 +52,70 @@ In your code:
...
 ```
+## Google Tag Manager
+
+You can load Smooch's web widget through Google Tag Manager. Simply connect to your Google Tag Manager account, go inside your container and follow the guide below. 
+
+### 1. Create a new Tag and give it a cute name like "Smooch"
+
+On your Google Tag Manager dashboard, create a new tag and rename it "Smooch".
+
+![GTM New Tag](/images/gtm_new_tag.png)
```
Comment (jpjoyal) [inline source/includes/_adding_smooch_js.md]: if you can make this not blurry that would be great.

File: source/includes/_adding_smooch_js.md [hunk 5]
```diff
@@ -52,3 +52,70 @@ In your code:
 ```javascript
 Smooch.init({appToken: 'your_app_token'});
 ```
+## Google Tag Manager
+
+You can load Smooch's web widget through Google Tag Manager. Simply connect to your Google Tag Manager account, go inside your container and follow the guide below.
```
Comment (alavers) [inline source/includes/_adding_smooch_js.md]: Google Tag Manager should be a hyperlink

**Comment by alavers** [comment]
Rest looks good

**Comment by Go-Fred** [comment]
Thanks guys, should be better now.

**Comment by Go-Fred** [comment]
@alavers Can I merge it?

**Pull request merged**